hbp_knowledge/er-stress/Karen2011.bel

```diff
@@ -147,29 +147,12 @@ p(HGNC:BAX) -> act(p(HGNC:ERN1), ma(kin))
 p(HGNC:BAK1) -> act(p(HGNC:ERN1), ma(kin))
 complex(p(HGNC:ERN1),p(HGNC:BAX),p(HGNC:BAK1))
 
-SET Evidence = "Protein tyrosine phosphatase
-1B (PTP1B), which is present mostly in the ER, also influences
-IRE1 activity. The absence of PTP1B caused impaired XBP1
-splicing, JNK phosphorylation and attenuated up-regulation of
-XBP1 target genes such as ER degradation enhancing alpha-mannosidase–
-like protein (EDEM) [12]."
-## Unable to find PTP1B in databases
```

Comment:
Google search for "Protein tyrosine phosphatase	1B (PTP1B)" linked to the right wikipedia page